apps/dashboard/.env

```
@@ -19,3 +19,4 @@ NEXT_PUBLIC_HEXCLAVE_DEBUGGER_ON_ASSERTION_ERROR=# set to true to open the debug
 HEXCLAVE_DISABLE_PLAN_LIMITS=# set to "true" to bypass enforcement of plan limits in the dashboard (seat-capacity gate on the projects page). Default unset/false preserves enforcement.
 HEXCLAVE_FEATUREBASE_JWT_SECRET=# used for Featurebase SSO, you probably won't have to set this
 HEXCLAVE_CHANGELOG_URL=# Used for raw github link to root changelog.md file.
+BYDEFAULT_INGEST_TOKEN=# optional ByDefault ingest token for AI-visibility tracking. Leave blank to disable tracking.
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BYDEFAULT_INGEST_TOKEN key should go before the HEXCLAVE_CHANGELOG_URL key

(UnorderedKey)